notebook/static/notebook/js/main.js

```diff
@@ -77,14 +77,6 @@ requirejs([
 
     // Pull typeahead from the global jquery object
     var typeahead = $.typeahead;
-    
-    try{
-        requirejs(['custom/custom'], function() {});
-        bidi.loadLocale();
-    } catch(err) {
-        console.log("Error processing custom.js. Logging and continuing");
-        console.warn(err);
-    }
```

Comment:
Important note:

I've moved these lines to the very end of the file, because the function `bidi.loadLocale` now reads metadata from the notebook and must be called after loading it. The problem is that I don't know the side effects of this change, especially because there's a line that loads a custom script and I don't know what it is at all.

Please just make sure I'm not breaking things here.

Reply:
`custom.js` is meant as a place for users to put arbitrary javascript to customise the notebook interface. Which probably means any change to how it's loaded is going to break something for someone, somewhere, but that can't really be helped.

I'm not sure why loading it is tied to bidi support, though. :-/

Reply:
Hi @m2-farzan, I suggest we break this up: leave the `custom.js` loading where it is and move the `bidi.locale()` handling to the end of the file.  This PR also needs a rebase.